CRM/Contribute/BAO/Product.php

```diff
@@ -111,20 +111,11 @@ public static function create($params) {
    * Delete premium Types.
    *
    * @param int $productID
-   *
+   * @deprecated
    * @throws \CRM_Core_Exception
    */
   public static function del($productID) {
-    //check dependencies
```

Comment:
OK - so to confirm this one it's a case of doing an r-run to verify that I can't delete a product if a premiums product entry exists.

I believe that the premiums product is the fact that someone 'bought' that product but I have to confirm that !

Reply:
Yes, something like that :)